### DISCUSSION TITLE
Additional Files added to project while upgrading to latest version (4.4.1 to 5.3.3).

### DISCUSSION THREAD

**Discussion by Rujwol** [twbs/bootstrap#40922] [Q&A] [answered]
Are all files essential? can we not include all/some files? what difference does it make?
What are the meaning of new files?
![image](https://github.com/user-attachments/assets/7999cffd-9d78-469a-90f2-ac67585e4377)
![image](https://github.com/user-attachments/assets/a415799e-3f93-4494-b30c-71296a0f8c08)

**Comment by julien-deramond** [comment, marked as answer]
They are not all essential, you'll have to choose the right ones depending on your needs. Most of the time `bootstrap.bundle.min.js` and `bootstrap.min.css` are enough as in our [Quick start](https://getbootstrap.com/docs/5.3/getting-started/introduction/#quick-start).
Otherwise, you'll need to check what to pick by looking at our [Contents page](https://getbootstrap.com/docs/5.3/getting-started/contents/).
